### PR TITLE
 Add eslint config to .lintstagedrc.js

### LINKED ISSUE #3646: Add eslint config to .lintstagedrc.js
Following these ticket / pull request:
- https://github.com/jhipster/jhipster-lite/issues/3424
- https://github.com/jhipster/jhipster-lite/pull/3645

When adding a front, we need to:
- check if `.lintstagedrc.js` exist
- then add `  '{,src/**/}*.{js,ts,tsx,vue}': ['eslint --fix'],`

The file should look like:

```javascript
module.exports = {
  '{src/**/,}*.{js,ts,tsx,vue}': ['eslint --fix'],
  '{src/**/,}*.{md,json,yml,html,css,scss,java,xml}': ['prettier --write'],
};
```

### PR DESCRIPTION
**Pull request by murdos**
Fixes #3646

### CODE REVIEW

File: src/test/java/tech/jhipster/lite/generator/client/angular/core/domain/AngularModuleFactoryTest.java [hunk 1]
```diff
@@ -57,6 +57,16 @@ void shouldCreateAngularModule() {
         "  \"jestSonar\": {\n    \"reportPath\": \"target/test-results/jest\",\n    \"reportFile\": \"TESTS-results-sonar.xml\"\n  },"
       )
       .and()
+      .hasFile(".lintstagedrc.js")
+      .containing(
+        """
+            module.exports = {
+              '{src/**/,}*.{js,ts,tsx,vue}': ['eslint --fix'],
```
Comment (Gnuk) [inline src/test/java/tech/jhipster/lite/generator/client/angular/core/domain/AngularModuleFactoryTest.java:64]: Maybe `'src/**/*.{js,ts,tsx,vue}'` is better (same for the other lines and files)

Comment (Gnuk) [reply src/test/java/tech/jhipster/lite/generator/client/angular/core/domain/AngularModuleFactoryTest.java:64]: (or it's to lint root files IDK?)

Comment (pascalgrimaud) [reply src/test/java/tech/jhipster/lite/generator/client/angular/core/domain/AngularModuleFactoryTest.java:64]: Yes it's for linting root file

**Comment by pascalgrimaud** [comment]
Thanks and congrats for your 1st PR @murdos

**Comment by murdos** [comment]
Bounty claimed: https://opencollective.com/generator-jhipster/expenses/98302

**Comment by pascalgrimaud** [comment]
@murdos : approved

**Pull request merged**